test/e2e-common.sh

```diff
@@ -18,6 +18,11 @@ source $(pwd)/vendor/knative.dev/hack/e2e-tests.sh
 source $(pwd)/hack/data-plane.sh
 source $(pwd)/hack/control-plane.sh
 
+# Latest release. If user does not supply this as a flag, the latest tagged release on the current branch will be used.
+readonly LATEST_RELEASE_VERSION="${LATEST_RELEASE_VERSION:-v1.0.0}"
```

Comment:
> I can see in the test logs that artifacts from https://storage.googleapis.com/knative-releases/eventing-kafka-broker/previous/v1.0.0 are used but this URL does't return anything. So, I'm wondering if the test was installing the right artifacts.
> Perhaps you could use the same as in Eventing? https://github.com/knative/eventing/blob/main/test/e2e-common.sh#L74

I tried using that function `latest_version` but it returned v0.27.0 which is not a real version and there are no artifacts for v0.27.0 in GCP.

Reply:
Regarding the URL:

https://storage.googleapis.com/knative-releases/eventing-kafka-broker/previous/v1.0.0 is the prefix, the resource URL is (for example): https://storage.googleapis.com/knative-releases/eventing-kafka-broker/previous/v1.0.0/eventing-kafka-broker.yaml

We append the YAML name here: https://github.com/knative-sandbox/eventing-kafka-broker/pull/617/files#diff-fbb940d8ac3d40d6c801e8a40575b2a3cd6d6e3f1d2d8f8122fbc2c438a6dccfR109-R111

Reply:
OK. Thanks. I'm confused by the versioning, TBH. Looking here: https://github.com/knative-sandbox/eventing-kafka-broker/releases , the version 0.27.0 is really not there but at the same time 1.0.2 is the latest. The version 1.0.0 as we have it in the scripts would go out-of-date soon probably.

Reply:
So, maybe it would make sense to temporarily override the function `latest_version` to work with knative-vXXX format of the tags or change tagging so that we use "v.X.Y.Z" instead of "knative-vX.Y.Z". I'm not sure why the tag naming scheme changed with 1.X releases. Possibly even adding tag alias `v1.0.2` for `knative-v1.0.2` would help and you could use the original `latest_version` function.

Reply:
I think I got why it wasn't working in our case and it is working in eventing core.

The eventing repository is using GH releases to download the previous artifact while I was using GCP.

Also, there is a [tag for 0.27.2](https://github.com/knative-sandbox/eventing-kafka-broker/releases/tag/v0.27.2) but the release name is 1.0.2.

Reply:
Let's see own it goes now.